modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -46,7 +46,7 @@ const RichText = ({
 		}
 	}, [editingLanguageId, editorRef]);
 
-	const currentValue = value ?? predefinedValue;
+	const currentValue = predefinedValue || value;
```

Comment:
I'm concerned about this fix, because to me it  seems that `predefinedValue` will always override the `value` here, so if the form's creator sets an `predefinedValue` the user won't be able to change its `value`.

Reply:
Hi @miltonmc , I also thought this would happen, but I tested it and I saw that after publishing a form, the `value` has the predefined value written in the builder, while the `predefinedValue` is empty.